.github/workflows/ci.yml

```diff
@@ -189,7 +189,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04, ubuntu-24.04, ubuntu-24.04-arm, macos-13, windows-2019, windows-2022]
-        go-version: ["1.23.8", "1.24.2"]
+        go-version: ["1.23.9", "1.24.3"]
```

Comment:
1.23.9 is meaningless as `go.mod` now requires 1.24

Reply:
you're right, but somehow the CI didn't fail 🤔. anyway removed 1.23.x

Reply:
CI didn't fail probably because it automatically updated Go 1.23 to 1.24.

We should have `GOTOOLCHAIN=local` to prevent this

Reply:
If the required modules is larger than the current go version in go.mod, it will attempt to download the latest go version and compile.

ref: https://go.dev/doc/toolchain#download